posts/2021/11/pydata-extensibility-vision.md

```diff
@@ -0,0 +1,410 @@
+<!--
+.. title: A vision for extensibility to GPU & distributed support for SciPy, scikit-learn, scikit-image and beyond
+.. slug: pydata-extensibility-vision
+.. date: 2021-11-15 10:00:00 UTC-07:00
+.. author: Ivan Yashchuk, Ralf Gommers
+.. tags: SciPy, NumPy, CuPy, scikit-learn, scikit-image, uarray, Array API
+.. category:
+.. link:
+.. description:
+.. type: text
+.. previewimage: /images/2021/11/nep-0047-library-dependencies.png
```

Comment:
I'm not sure how whether it's supposed to work, but this field does nothing at the moment.

Reply:
I believe that you'll only see that in things like the Twitter link preview thumbnail.